sensor_msgs_py/setup.py

```diff
@@ -0,0 +1,21 @@
+from setuptools import setup
+
+package_name = 'sensor_msgs_py'
+
+setup(
+    name=package_name,
+    version='2.1.0',
+    packages=[package_name],
+    data_files=[
+        ('share/ament_index/resource_index/packages',
+            ['resource/' + package_name]),
+        ('share/' + package_name, ['package.xml']),
+    ],
+    install_requires=['setuptools'],
+    zip_safe=True,
+    maintainer='Sebastian Grans',
+    maintainer_email='sebastian.grans@gmail.com',
```

Comment:
Same thing about the maintainers here; I'll suggest making the maintainers the same as the package.

Reply:
@christophebedard i didnt do this change [on my PR](https://github.com/SebastianGrans/common_interfaces/pull/4) because i figured we should probably get @wjwwood OK first.

Reply:
As far as I understand, `setup.py` only supports one maintainer. So I added Michael Carroll as the maintainer.